src/__tests__/options2.test.ts

```diff
@@ -0,0 +1,54 @@
+import { remark } from "remark";
+import type { Processor } from "unified";
+import type { VFileCompatible } from "vfile";
+import gfm from "remark-gfm";
+import remark2rehype from "remark-rehype";
+import stringify from "rehype-stringify";
+
+import plugin, {type CustomContainerOptions} from "..";
+
+const compiler: Processor = remark()
+  .use(gfm)
+  .use(plugin, {
+    additionalProperties: (className, title) => {}
+  } as CustomContainerOptions)
```

Comment:
may I ask you to add a test without any options, just confirm `use(plugin)` works well without breaking change?